cabal.project

```diff
@@ -1 +1,5 @@
-packages: */*.cabal
```

Comment:
Actually, the wildcard works well, doesn't it? I'd rather leave it if it doesn't make any trouble.